app/controllers/catalog_controller.rb

```diff
@@ -154,13 +154,13 @@ def enforce_show_permissions(_opts = {})
     # config.default_solr_params[:'facet.field'] =
     #   config.facet_fields.select { |_k, v| v[:show] != false }.keys
 
-    # solr fields to be displayed in the index (search results) view
+    # Solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
     config.add_index_field solr_name("work_type_label", :stored_searchable),
                            label: "Format"
 
     config.add_index_field solr_name("collection_label", :symbol),
-                           label: "Collection"
+                           label: "In Collection"
```

Comment:
@chrissyrissmeyer is this OK?  I figured since Collections themselves will be part of the search results list, saying `In Collection` would be less ambiguous.